tests/tests_app/components/multi_node/test_lite.py

```diff
@@ -0,0 +1,110 @@
+import os
+from copy import deepcopy
+from functools import partial
+from unittest import mock
+
+import pytest
+from lightning_utilities.core.imports import module_available
+from tests_app.helpers.utils import no_warning_call
+
+import lightning_lite as ll
```

Comment:
Try to change it to lightning.lite. The tracer only patches these imports (could be considered a bug)